.github/workflows/build.yml

```diff
@@ -39,13 +42,64 @@ jobs:
         make -f certifier_tests.mak clean
 
         make -j2 -f certifier.mak
+
+        # We need to clean here, otherwise make certifier_tests.mak will run
+        # into some protobuf-related errors. But this 'clean' will also
+        # rm ../libcertifier_framework.so, so it will need to be re-built.)
         make -f certifier.mak clean
 
         make -j2 -f certifier_tests.mak
         ./certifier_tests.exe
 
+        # Re-build ...; otherwise pylint will complain below.
+        make -f certifier.mak ../libcertifier_framework.so
```

Comment:
Minor wart in our make systems ... difficult to resolve otherwise, w/o extra effort.

This should be ok for now..

Reply:
Note to myself: Update comment to read: 

"Rebuild shared library as previous make clean deleted it. Otherwise, pylint, below, will complain that .so was not found."

Reply:
Updated comment.